aio/content/navigation.json

```diff
@@ -357,8 +357,8 @@
             },
             {
               "url": "guide/upgrade-lazy",
-              "title": "Lazy-loading Angular into AngularJS",
-              "tooltip": "Upgrade from AngularJS to Angular in a more flexible way."
+              "title": "Hybrid Angular/AngularJS Apps",
```

Comment:
Yeah, they overlap some. This doc seems to be about setting up this hybrid scenario with that being the end state (of course, in the grand scheme it's only an intermediate step), and in the previous doc, the hybrid construct is covered, but isn't the main focus of the guide....right? I could be missing a nuance. As far as the title goes, the reason I changed it was because the doc doesn't explicitly talk about lazy loading. I'm scratching my head thinking how to make this title clearer. Am I understanding the difference between the two? If I can that difference, maybe I can come up with a clearer title. But then, maybe there's a way we could point out how this is lazy loading?